src/Template/Layout/mailer.ctp

```diff
@@ -0,0 +1,13 @@
+<?php $this->extend('toolbar') ?>
+
+<div style="height:calc(100vh);overflow-y:scroll">
+    <?php if (empty($noHeader)) : ?>
+        <h2 class="panel-title">
+            <?= isset($title) ? $title : "Mailer Previews 📮 " ?>
```

Comment:
Should we use HTML codepoints instead of raw bytes for emoji? Keeping the source ascii compatible has some nice benefits.